docs/ref/models/querysets.txt

```diff
@@ -694,13 +694,15 @@ objects representing all available dates of a particular kind within the
 contents of the ``QuerySet``.
 
 ``field`` should be the name of a ``DateField`` of your model.
-``kind`` should be either ``"year"``, ``"month"`` or ``"day"``. Each
-``datetime.date`` object in the result list is "truncated" to the given
+``kind`` should be either ``"year"``, ``"month"``, ``"week"`` or ``"day"``.
+Each ``datetime.date`` object in the result list is "truncated" to the given
 ``type``.
 
 * ``"year"`` returns a list of all distinct year values for the field.
 * ``"month"`` returns a list of all distinct year/month values for the
   field.
+* ``"week"`` returns a list of all distinct year/month/day values for the
```

Comment:
Should this be different from the description of "day"? Does the "day" in the return datetime actually signify "week"?

Reply:
It returns a `date time.date` object, but all dates will be Mondays. I guess I should have specified that.

Reply:
It seems suspicious because the new test in tests/dates/tests.py returns 7/25 and 7/26 -- they both can't be Monday, right?